usr/share/rear/conf/default.conf

```diff
@@ -1416,15 +1442,8 @@ BACKUP_PROG_CRYPT_ENABLED="false"
 # cf. the reasoning about SSH_UNPROTECTED_PRIVATE_KEYS below
 # and see https://github.com/rear/rear/issues/2155
 # Therefore BACKUP_PROG_CRYPT_KEY must be manually set before running "rear recover".
-# Avoid that the BACKUP_PROG_CRYPT_KEY value is shown when usr/sbin/rear was called with 'set -x'
-# for debugging usr/sbin/rear cf. https://github.com/rear/rear/issues/2144#issuecomment-493908133
-# and see the comment of the UserInput function in lib/_input-output-functions.sh
-# how to keep things confidential when usr/sbin/rear is run in debugscript mode.
-# In debugscript mode only scripts sourced by the Source function in lib/framework-functions.sh
-# are run with 'set -x' but default.conf is sourced by usr/sbin/rear directly.
-# Nevertheless the confidential way via { confidential_command ; } 2>/dev/null
-# is also used here to provide a template how to do it properly in etc/rear/local.conf:
```

Comment:
Hello @jsmeix , `default.conf` is sourced by `usr/sbin/rear` directly (using`source`) while `{site,local}.conf` are sourced using `Source` (which can set `-x`). The comment thus looks correct and might be good to preserve it.
Wouldn't it be better though to avoid sourcing `{site,local}.conf` with `Source` and thus avoid the possibility of `-x` being set and exposing secret values entirely, without having to protect every private variable (and thus having to remember to do it)? Seems like a cleaner solution to me, WDYT?

Reply:
@pcahyna thank you for your review.
Via
https://github.com/rear/rear/pull/2982/commits/3c580c5c981c7ea2c4c14ca28e9ffed28761d1b2
I keep the information that default.conf is sourced
by usr/sbin/rear directly (using 'source')
while {site,local}.conf are sourced using Source()
which can 'set -x'.